src/button/_button.scss

```diff
@@ -52,126 +52,126 @@
   &.btn-outline-warning {
     &:disabled, &.disabled {
       & .spinner-border {
-        color: tokens.$color-white;
+        color: tokens.$color-bg-white;
       }
     }
   }
 
   &.btn-primary {
     &:focus {
-      background-color: tokens.$color-primary-500;
-      border-color: tokens.$color-primary-500;
+      background-color: tokens.$color-default-500;
+      border-color: tokens.$color-default-500;
     }
 
     &:hover {
-      background-color: tokens.$color-primary-600;
-      border-color: tokens.$color-primary-600;  
+      background-color: tokens.$color-default-600;
+      border-color: tokens.$color-default-600;  
     }
 
     &:active, &.active {
-      background-color: tokens.$color-primary-600;
-      border-color: tokens.$color-primary-600;  
+      background-color: tokens.$color-default-600;
```

Comment:
needs to be `tokens.$color-default-700`

Reply:
Should the border-color remain 600, or should it also be 700?